index.d.ts

```diff
@@ -1751,7 +1751,7 @@ export type PackageMetadata = {
 	/**
 	 * Files to preload
 	 */
-	files?: string[];
+	files?: (object | string)[];
```

Comment:
I don't think this is valid :thinking: 

```
files?: object[] | string[];
```

Reply:
I think `files?: object[] | string[];` means it should be an array of objects OR an array of strings. But, `(object | string)[]` means it's an array of objects or strings. I'm not an expert in Typescript but let me know if I'm wrong.

Reply:
Ah, yes. I think I prefer the following syntax when defining a union type like this `Array<object | string>` :thinking: